plugins/ufm_events_grafana_dashboard_plugin/scripts/entrypoint.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+# @author: Anan Al-Aghbar
+# @date:   Aug 25, 2024
+#
+
```

Comment:
Every place where accessing a var, please use "${var_name}" and not "$var_name"
It is much saferer